### PR TITLE
Universal/split project correct shader stripping

### PR DESCRIPTION
**Pull request by Nzollner**
# Purpose of this PR
This PR ensures the post-processing and mixed lighting shader variants are removed from UniversalGraphicsTest_Foundation.
By doing this will we need less time to run the foundation tests.

# Testing status
I have tested if the tests pass for both foundation and post-processing since I have changed objects in the projects.
I have also tested if the post-processing and mixed lighting shader variants were present after building. Which they shouldn't.

# Comments to reviewers
The tool I used to check shader variants were: 
https://github.com/Unity-Technologies/Graphics-QA-Scripts/tree/master/Shader_Variant_Tool

### CODE REVIEW

**Comment by Nzollner** [comment]
Should there be a changelog for this?

**Comment by Nzollner** [comment]
Should there be backlogs for this?

File: TestProjects/UniversalGraphicsTest_Foundation/Assets/Scenes/056_2D_Lights/New 2D Renderer Data.asset [hunk 1]
```diff
@@ -50,7 +50,7 @@ MonoBehaviour:
     type: 3}
   m_FallbackErrorShader: {fileID: 4800000, guid: e6e9a19c3678ded42a3bc431ebef7dbd,
     type: 3}
-  m_PostProcessData: {fileID: 11400000, guid: 41439944d30ece34e96484bdb6645b55, type: 2}
+  m_PostProcessData: {fileID: 0}
   m_FallOffLookup: {fileID: 2800000, guid: 5688ab254e4c0634f8d6c8e0792331ca, type: 3}
   m_DefaultMaterialType: 0
   m_DefaultCustomMaterial: {fileID: 2100000, guid: a97c105638bdf8b4a8650670310a4cd3,
```
Comment (ellioman) [inline TestProjects/UniversalGraphicsTest_Foundation/Assets/Scenes/056_2D_Lights/New 2D Renderer Data.asset:56]: Can't we kill this entire renderer? I don't think the 056 test is in this project.

File: TestProjects/UniversalGraphicsTest_Foundation/Assets/Scenes/120_PostProcessingRenderUICustomRenderer/120_PostProcessingRenderUICustomRenderer_Renderer.asset [hunk 2]
```diff
@@ -15,7 +15,7 @@ MonoBehaviour:
   m_RendererFeatures:
   - {fileID: 4450485241359397263}
   m_RendererFeatureMap: 
-  postProcessData: {fileID: 11400000, guid: 41439944d30ece34e96484bdb6645b55, type: 2}
+  postProcessData: {fileID: 0}
   xrSystemData: {fileID: 11400000, guid: 60e1133243b97e347b653163a8c01b64, type: 2}
   shaders:
     blitPS: {fileID: 4800000, guid: c17132b1f77d20942aa75f8429c0f8bc, type: 3}
```
Comment (ellioman) [inline TestProjects/UniversalGraphicsTest_Foundation/Assets/Scenes/120_PostProcessingRenderUICustomRenderer/120_PostProcessingRenderUICustomRenderer_Renderer.asset:21]: Is this renderer being used in this project?

**Comment by ellioman** [comment]
> Should there be a changelog for this?
Nope, not needed for this.

File: TestProjects/UniversalGraphicsTest_Foundation/Assets/Scenes/120_PostProcessingRenderUICustomRenderer/120_PostProcessingRenderUICustomRenderer_Renderer.asset (hunk 2 above)
Comment (Nzollner) [reply TestProjects/UniversalGraphicsTest_Foundation/Assets/Scenes/120_PostProcessingRenderUICustomRenderer/120_PostProcessingRenderUICustomRenderer_Renderer.asset:21]: used inthe scenes:
120
121
122

Comment (Nzollner) [reply TestProjects/UniversalGraphicsTest_Foundation/Assets/Scenes/120_PostProcessingRenderUICustomRenderer/120_PostProcessingRenderUICustomRenderer_Renderer.asset]: removed the renderer from those scenes using the default renderer

File: TestProjects/UniversalGraphicsTest_Foundation/Assets/Scenes/056_2D_Lights/New 2D Renderer Data.asset (hunk 1 above)
Comment (Nzollner) [reply TestProjects/UniversalGraphicsTest_Foundation/Assets/Scenes/056_2D_Lights/New 2D Renderer Data.asset:56]: Had to move this renderer to common assets for the 2d tests to work

**Comment by Nzollner** [comment]
@ellioman is looking into the split tree shader variants for the post-processing split project.

**Comment by ellioman** [comment]
> @ellioman is looking into the split tree shader variants for the post-processing split project.

Yes, #3174 will contain various improvements to the test scenes and split it so the Terrain Project will test Terrain + Terrain Details + SpeedTrees with SSAO and Foundation will test URP + ShaderGraph.

**Comment by Nzollner** [comment]
Lighting has errors on Yamato. I have split the PR into 2 parts because of that:
https://github.com/Unity-Technologies/Graphics/pull/3724
https://github.com/Unity-Technologies/Graphics/pull/3723

**Pull request closed without merging**